src/Components/__stories__/ArtworkMetadata.story.tsx

```diff
@@ -0,0 +1,27 @@
+import { storiesOf } from "@storybook/react"
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import ArtworkMetadata from "../Artwork/Metadata"
+
+import { artsyNetworkLayer } from "../../Relay/config"
+import ArtworkQueryConfig from "../../Relay/Queries/Artwork"
+
+function ArtworkMetadataGenerator(data) {
+  return <ArtworkMetadata {...data} extended />
+}
+
+function ArtworkExample(props: { artworkID: string }) {
+  Relay.injectNetworkLayer(artsyNetworkLayer())
+  return (
+    <Relay.RootContainer
+      renderFetched={ArtworkMetadataGenerator}
+      Component={ArtworkMetadata}
+      route={new ArtworkQueryConfig({ artworkID: props.artworkID })}
+    />
+  )
+}
+
+storiesOf("Components/Artwork/Metadata", module)
+  .add("A NFS artwork", () => <ArtworkExample artworkID="andy-warhol-skull" />)
```

Comment:
Goal is to add a lot of cases here (probably separated by partner)? It should be pretty quick to add. So far these are looking good!

<img width="134" alt="screen shot 2017-10-26 at 12 32 51 pm" src="https://user-images.githubusercontent.com/1457859/32065119-ce4d5b6e-ba49-11e7-8019-0c737ee6bf2f.png">

Reply:
I’m not a big fan of shortening text, especially not when used in a UI. Was there a reason to do that here (“NFS”) other than not having to type more?

Reply:
Nope, I guess I've seen NFS used sometimes as an abbreviation (at Artsy), but no special reason whatsoever.